src/Bicep.Cli/CliResources.resx

```diff
@@ -142,7 +142,7 @@
     <comment>{0} input file path</comment>
   </data>
   <data name="ExperimentalFeaturesDisclaimerMessage" xml:space="preserve">
-    <value>WARNING: The following experimental Bicep features have been enabled: {0}. Experimental features should be enabled for testing purposes only. Do not enable these settings for any production usage, or you may be unexpectedly broken at any time!</value>
+    <value>WARNING: The following experimental Bicep features have been enabled: {0}. Experimental features should be enabled for testing purposes only, as there are no guarantee about the quality or stability of these features. Do not enable these settings for any production usage, or your production environment may be subject to breaking.</value>

```

Comment:
```suggestion
    <value>WARNING: The following experimental Bicep features have been enabled: {0}. Experimental features should be enabled for testing purposes only, as there are no guarantees about the quality or stability of these features. Do not enable these settings for any production usage, or your production environment may behave in an unexpected manner or stop working at any time.</value>
```

"Subject to breaking" sounds a bit formal and vague to me. What about something like the above?

Reply:
As discussed, I'm using the text from #11953, but will correct the grammar error.